code/elemental/ElementalSearchableFieldsBlock.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace SilverStripe\FrameworkTest\Elemental\Model;
+
+use DNADesign\Elemental\Models\BaseElement;
+use SilverStripe\Forms\SearchableDropdownField;
+use SilverStripe\Forms\SearchableMultiDropdownField;
+
+class ElementalSearchableFieldsBlock extends BaseElement
+{
+    private static $table_name = 'ElementalSearchableFieldsBlock';
+
+    private static $has_one = [
+        'ElementalSearchableFieldsObject' => ElementalSearchableFieldsObject::class,
+    ];
+
+    private static $has_many = [
+        'ElementalSearchableFieldsObjects' => ElementalSearchableFieldsObject::class,
+    ];
```

Comment:
Just use something we already know exists already, like `Member` or `Page` - that way we don't need the extra `ElementalSearchableFieldsObject` class and it makes this easier to test locally since we have a UI to create objects of those classes already.

Reply:
I choose not to use existing ones because for the has_many relation I need a corresponding has_one relation to ElementalSearchableFieldBlock. I don't want to add that to Member or Page

Reply:
Use a many_many instead?

Reply:
Updated